apps/freelance/src/redux/services/authApi.ts

```diff
@@ -0,0 +1,18 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const authApi = createApi({
+  reducerPath: 'authReducer',
+  baseQuery: fetchBaseQuery({ baseUrl: 'http://localhost:3000/auth' }),
```

Comment:
no links in the code Never, or envs or constants

Reply:
updated to constants